client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/test/index.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/element';
+import { render } from '@testing-library/react';
+import { useSelect } from '@wordpress/data';
+import userEvent from '@testing-library/user-event';
+import { pluginNames } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import { SelectiveExtensionsBundle } from '../';
+
+jest.mock( '../../app-illustration', () => ( {
+	AppIllustration: jest.fn().mockReturnValue( '[illustration]' ),
+} ) );
+
+jest.mock( '@wordpress/data', () => ( {
+	...jest.requireActual( '@wordpress/data' ),
+	useSelect: jest.fn(),
+	useDispatch: jest.fn().mockImplementation( () => ( {
+		updateOptions: jest.fn(),
+		installAndActivatePlugins: jest.fn(),
+	} ) ),
+} ) );
+
+jest.mock( '@woocommerce/data', () => ( {
+	pluginNames: {
+		'woocommerce-payments': 'WooCommerce Payments',
+		mailpoet: 'Mailpoet',
+		random: 'Random',
+		'google-listings-and-ads': 'Google Listings and Ads',
+	},
+} ) );
+
+const freeExtensions = [
+	{
+		key: 'basics',
+		title: 'Get the basics',
+		plugins: [
+			{
+				key: 'woocommerce-payments',
+				description: 'WC Pay Description',
+				is_visible: true,
+				is_installed: true,
+			},
+			{
+				key: 'mailpoet',
+				description: 'Mailpoet Description',
+				manage_url: 'admin.php?page=mailpoet-newsletters',
+				is_visible: true,
+				is_installed: true,
+			},
+		],
+	},
+	{
+		key: 'reach',
+		title: 'Reach out to customers',
+		plugins: [
+			{
+				key: 'random',
+				name: 'Random',
+				description: 'Random description',
+				manage_url: 'admin.php?page=mailpoet-newsletters',
+				is_visible: true,
+				is_installed: true,
+			},
+		],
+	},
+	{
+		key: 'grow',
+		title: 'Grow your store',
+		plugins: [
+			{
+				key: 'google-listings-and-ads',
+				name: 'Google Ads & Marketing by Kliken',
+				description: 'Google Description',
+				manage_url: 'admin.php?page=wc-admin&path=%2Fgoogle%2Fstart',
+				is_visible: true,
+				is_installed: false,
+			},
+		],
+	},
+];
+
+describe( 'Selective extensions bundle', () => {
+	it( 'should list installable free extensions in footer only basics', () => {
+		useSelect.mockReturnValue( {
+			freeExtensions: freeExtensions,
```

Comment:
@louwie17 Note that there's a lint error here

Reply:
Thanks, I fixed those in https://github.com/woocommerce/woocommerce-admin/pull/7585/commits/398f46a8726ace7f7c13440b6bef6985dff495ed